packages/env/lib/config.js

```diff
@@ -257,9 +257,25 @@ function parseSourceString( sourceString, { workDirectoryPath } ) {
 		};
 	}
 
+	const wpOrgFields = sourceString.match(
+		/^https?:\/\/downloads\.wordpress\.org\/(plugin|theme)\/([^\s\.]*)([^\s]*)?\.zip$/
+	);
+	if ( wpOrgFields ) {
+		return {
+			type: 'zip',
+			url: sourceString,
+			path: path.resolve(
+				workDirectoryPath,
+				encodeURIComponent( wpOrgFields[ 2 ] )
+			),
+			basename: encodeURIComponent( wpOrgFields[ 2 ] ),
```

Comment:
should we just make this apply to the `zipFields` thing below, and use the second index instead of the first? I guess a reason to not do that would be to allow zip sources from elsewhere to continue working :)